src/incubator/hooks/useHiddenLocation.ts

```diff
@@ -24,10 +22,10 @@ export default function useHiddenLocation<T extends View>(props: HiddenLocationP
     width = Constants.screenWidth,
     height = Constants.windowHeight,
     isDefault = true
-  }) => {
+  }): HiddenLocation => {
     return {
-      top: -y - height,
-      bottom: Constants.windowHeight - y,
+      up: -y - height,
+      down: Constants.windowHeight - y,
```

Comment:
Why did u change to up/down?

Reply:
So it will be backwards compatible with the old enum and remove some awkward "translations" between the two enums.

Reply:
I see.. bummer I prefer top/bottom on up/down
If the components are in Incubator I don't mind changing it, I don't consider it a breaking change

Reply:
It'll require a migration (public, private, apps...) because we're using up\down [here](https://github.com/wix/react-native-ui-lib/blob/master/src/components/panningViews/panningProvider.tsx#L4-L13).
Or some code I removed to translate it (and you did not like, if I recall correctly).
I figured we can use the old convention, at least for now, and a migration can be done in any time.

Reply:
ok 👍